build/model_dist.py

```diff
@@ -108,6 +111,7 @@ def forward(self, x: Tensor, input_pos: Optional[Tensor] = None) -> Tensor:
 
         if self.stage_idx == self.n_stages - 1:
             x = self.norm(x)
+            logger.info(f"final stage output info: {x.dtype=}, {x.shape=}, {x.device=}\n {self.output.weight.dtype=}, {self.output.weight.shape=},")
```

Comment:
maybe comment out the logging? 
torch compile or export may graph break here if without special flags.

Reply:
thanks for catching.  I didn't intend to leave this in, and have removed.